code/modules/surgery/experimental_dissection.dm

```diff
@@ -45,31 +45,34 @@
 	user.visible_message("<span class='notice'>[user] starts dissecting [target].</span>", "<span class='notice'>You start dissecting [target].</span>")
 
 /datum/surgery_step/dissection/proc/check_value(mob/living/target, datum/surgery/advanced/experimental_dissection/ED)
-	var/cost = BASE_HUMAN_REWARD
+	var/cost = 0
 	var/multi_surgery_adjust = 0
 
 	//determine bonus applied
+	var/list/mob_mult_list = list(/mob/living/simple_animal/hostile/asteroid/elite = 6,
```

Comment:
```suggestion
	var/static/list/mob_mult_list = list(/mob/living/simple_animal/hostile/asteroid/elite = 6,
``` 
just so it doesn't need to get re-created every time, it won't matter since it's just a surgerybut it can't hurt